flow/layers/image_filter_layer.cc

```diff
@@ -56,7 +57,7 @@ void ImageFilterLayer::Preroll(PrerollContext* context,
   }
 
   const SkIRect filter_input_bounds = child_bounds.roundOut();
-  SkIRect filter_output_bounds = filter_->filterBounds(
+  SkIRect filter_output_bounds = filter_->skia_object()->filterBounds(
```

Comment:
DlImageFilter provides this bounds operation directly, no need to invoke it on skia_object.

Reply:
Which one is that?